recipes/onetbb/all/patches/pkg-search-module-use-imported-target.diff

```diff
@@ -0,0 +1,97 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
I see this patch is actually a collection of small patches already in the upstream. Could you please, use the official patches, pointing there urls as `patch_source` in conandata.yml? So it will be much easier to track their source.

Reply:
You know, this is not a single patch and they are made not one after another, so in order to use official patches, I need to refer to multiple ones (tree of patches) and they contain changes not related to this PR.

Reply:
Could you please at least, refer them in conandata.yml? 

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patches

```
patch_sources:
  - "https://github.com/...."
  - "https://github.com/...."
  - "https://github.com/...."
```

Reply:
done.